content/source/docs/enterprise/install/uninstall.html.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: "enterprise"
+page_title: "Uninstall - Terraform Enterprise"
+---
+
+# Uninstall Terraform Enterprise
+
+If you installed Terraform Enterprise on VMWare instances, you may not be able to easily request new virtual machines for a broken or corrupted installation. Instead, you can use the `uninstall` script to remove Terraform Enterprise and all its services (excluding Docker) from a system.
+
+Please contact [support][support] with questions or issues.
+
+## While the Script Runs
+
+After you initiate `uninstall`:
+
+1. Enter `yes` when asked if you want to continue with the installation.
+
+![Uninstall - Do you want to continue?][uninstall-prompt]
+
+   The script stops the Replicated services, removes the Docker containers, and removes replicated executables and configuration files from the system.  
+
+![Uninstall - Application removed][uninstall-uninstalled]
+2. Choose an option to prune Docker volumes:  
+  - Select `Prune all Docker volumes` if you only use this system for Terraform Enterprise.  
+  - Select `Prune only application Docker volumes` to prune only Replicated and Hashicorp Docker volumes and leave the rest intact.  
+  - Select `Skip this step` to leave all Docker volumes intact.  
+![Uninstall - Prune all Docker volumes][uninstall-prune-all]
+
+3. The script removes dangling Docker volumes and the Docker networks that were created for the application. Terraform Enterprise has been removed from the system.
+
+![uninstall-cleanup][uninstall-cleanup]
+
+~> **Important**: This script does not touch the mounted disk path, so you will need to manually clean it up if necessary.
+
+## Run the Uninstaller
```

Comment:
To me it makes sense for this to come first on the page before we talk about what happens while the script runs, but open to pushback if we feel strongly that it should be second.

Reply:
I'm open to this as well.